# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/atomic.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./include/asm-i386/atomic.h.orig	2005-03-10 22:12:23.104674184 -0300
+++ ./include/asm-i386/atomic.h	2005-03-10 22:14:38.358112536 -0300
@@ -3,7 +3,9 @@
 
 #include <linux/config.h>
 #include <linux/compiler.h>
-#include <asm/processor.h>
+#ifdef __KERNEL__
+# include <asm/processor.h>
+#endif
 
 /*
  * Atomic operations that C can't guarantee us.  Useful for
